src/main/java/uk/gov/cshr/civilservant/controller/CivilServantController.java

```diff
@@ -105,6 +114,83 @@ public CivilServantController(LineManagerService lineManagerService, CivilServan
         return ResponseEntity.unprocessableEntity().build();
     }
 
+    @GetMapping("/org")
```

Comment:
These organisation methods should maybe be moved to another class. It make sense for `CivilServantController` to only contain methods pertaining to the CivilServant domain objects.

Reply:
As discussed, this is parked until we have more time.